Re-styling of the docs, more content, sidebar menu

richorama opened the pull request:
Added 1.1.0 release
Adjusted the content on the landing page
Reformatted the nuget package links (to make it easier to copy and paste them)

There's still lots more work to do!

jthelin commented:
I added a no-op `Build.cmd` file onto the gh-pages thread so we wouldn't get docs changes showing up as "failed" build checks by Jenkins. 
Did you intended to delete that file in this change set, @richorama ?

```
08:01:03 d:\j\workspace\dotnet_orleans\innerloop_prtest>call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\VsDevCmd.bat"   && Build.cmd   && Test.cmd
08:01:04 'Build.cmd' is not recognized as an internal or external command, operable program or batch file.
```

jthelin commented on NuGets.md:
```diff
@@ -5,35 +5,120 @@ title: Orleans NuGet Packages
 
 
-### Orleans NuGet packages as of [v1.0.9](https://github.com/dotnet/orleans/releases/tag/v1.0.9)
+## Orleans NuGet packages as of [v1.0.9](https://github.com/dotnet/orleans/releases/tag/v1.0.9)
```
Should we say "since 1.0.9" here, and just point to the releases page?
https://github.com/dotnet/orleans/releases

Or even just drop the version number entirely and by assumption talk about "current"?

richorama commented:
Ah, I deleted the file because it didn't do anything!

-----Original Message-----
From: "Jorgen Thelin" <notifications@github.com>
Sent: ‎21/‎12/‎2015 16:33
To: "dotnet/orleans" <orleans@noreply.github.com>
Cc: "Richard Astbury" <richard.astbury@gmail.com>
Subject: Re: [orleans] Re-styling of the docs, more content, sidebar menu(#1191)

I added a no-op Build.cmd file onto the gh-pages thread so we wouldn't get docs changes showing up as "failed" build checks by Jenkins. 
Did you intended to delete that file in this change set, @richorama ?
08:01:03 d:\j\workspace\dotnet_orleans\innerloop_prtest>call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\VsDevCmd.bat"   && Build.cmd   && Test.cmd
08:01:04 'Build.cmd' is not recognized as an internal or external command, operable program or batch file.
—
Reply to this email directly or view it on GitHub.

jthelin commented:
> Ah, I deleted the file because it didn't do anything!

;) 

Both Build.cmd and Test.cmd on the gh-pages branch are "no-op" scripts to keep Jenkins happy when building docs changes. The DotNet Jenkins guys haven't yet found a way for us to use different build configs for some branches (gh-pages specifically) so adding no-op scripts was my kludgy fix.

jthelin commented:
Need to restore Test.cmd too. You deleted that too :cry:

jthelin commented:
Overall the UI changes look good to me.

It is hard to tell for sure without online viewable copy, and i could not work out easy way to do that.

I suggest we leave this PR open for a short while so other have a change to look, but then maybe merge and recheck how the web site looks.

richorama commented:
Look at http://Richorama.github.io/orleans for a viewable copy...

-----Original Message-----
From: "Jorgen Thelin" <notifications@github.com>
Sent: ‎21/‎12/‎2015 16:48
To: "dotnet/orleans" <orleans@noreply.github.com>
Cc: "Richard Astbury" <richard.astbury@gmail.com>
Subject: Re: [orleans] Re-styling of the docs, more content, sidebar menu(#1191)

Overall the UI changes look good to me.
It is hard to tell for sure without online viewable copy, and i could not work out easy way to do that.
I suggest we leave this PR open for a short while so other have a change to look, but then maybe merge and recheck how the web site looks.
—
Reply to this email directly or view it on GitHub.

gabikliot commented:
OMG!
This looks SO much better!

sergeybykov commented:
:+1: Nice and clean. I think we should get rid of "Project" and go with "Microsoft Orleans" from now on.
